server/game/cards/08.5-TFM/TheFaithMilitant.js

```diff
@@ -0,0 +1,43 @@
+const AgendaCard = require('../../agendacard.js');
+
+class TheFaithMilitant extends AgendaCard {
+    setupCardAbilities(ability) {
+
+        this.persistentEffect({
+            match: card =>
+                !card.hasTrait('The Seven') &&
+                ['character', 'location'].includes(card.getType()) &&
+                card.controller === this.controller,
```

Comment:
Not an issue, but also not needed as persistentEffects default to the current controller.

Reply:
Thanks for the review. All comments addressed except this one. I was aware of the default controller, but I quite like when there is a 1-to-1 mapping between card text ("you control") and card implementation (the "===" requirement). This probably deserves a more broad discussion about whether we prefer to have shorter card implementations or more explicit ones.

Reply:
👍